frontend/components/Goals/index.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import image from './images/Squiggles Monochromatic.png';
+import ContentLoader from 'react-content-loader';
+import './style.css';
```

Comment:
follow the project structure by adding the CSS files in the styles folder located at the root directory, 
then import it in the index.css file